blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticIO.java

```diff
@@ -132,7 +122,7 @@ private static String getUnit(Metric metric) {
         return metric.getUnit();
     }
 
```

Comment:
Nevermind, i am going to leave it as internal so we can unit test it by itself.  Adding test for it in ElasticIOTest, which should also be in the com.rackspacecloud.blueflood.io package (and not service).